view/next-project/src/components/sponsorstyles/DeleteModal.tsx

```diff
@@ -1,10 +1,9 @@
 import { useRouter } from 'next/router';
 import React, { Dispatch, FC, SetStateAction } from 'react';
-
 import { del } from '@api/api_methods';
 import { Modal, CloseButton, OutlinePrimaryButton, PrimaryButton } from '@components/common';
 
-interface ModalProps {
+export interface ModalProps {
```

Comment:
```suggestion
interface ModalProps {
```
ModalPropsは使っていなので、exportしなくていいかと思います！